### PR TITLE
Add blog-post: Calls for GSoC 2023 projects.

### PR DESCRIPTION
**Pull request by yawen-guan**

### CODE REVIEW

**Review by anatoliykmetyuk — APPROVED** [review]

File: _posts/2023-01-23-gsoc-2023.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+category: blog-detail
+post-type: blog
+by: Scala Center
+title: "Calls for Scala GSoC 2023 projects!"
+---
+
+Calls for open source Scala projects for [Google Summer of Code 2023](https://summerofcode.withgoogle.com/)!
+
+After [the success of last year's edition](https://scala-lang.org/blog/2022/11/15/gsoc-report.html), the Scala Center is applying to participate GSoC 2023. We are currently reaching out to mentors to assemble a Project List for Scala GSoC 2023. This is a great opportunity for open source Scala project maintainers to get some fresh hands on their projects and help out new contributors at the very beginning of their Scala career!
```
Comment (SethTisue) [inline _posts/2023-01-23-gsoc-2023.md:10]: s/participate/participate in/

File: _posts/2023-01-23-gsoc-2023.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+category: blog-detail
+post-type: blog
+by: Scala Center
+title: "Calls for Scala GSoC 2023 projects!"
+---
+
+Calls for open source Scala projects for [Google Summer of Code 2023](https://summerofcode.withgoogle.com/)!
```
Comment (SethTisue) [inline _posts/2023-01-23-gsoc-2023.md:8]: I guess you're assuming it's common knowledge what GSoC even is — or that people will follow the link if they don't know. Regardless, I'd suggest including some brief wording like: "GSoC is an internship program where Google funds students to work on open-source projects."

Comment (SethTisue) [reply _posts/2023-01-23-gsoc-2023.md:8]: s/Calls for/Call for/

**Review by SethTisue — APPROVED** [review]
fine as is, but I've made a few minor suggestions anyway

File: _posts/2023-01-23-gsoc-2023.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+After [the success of last year's edition](https://scala-lang.org/blog/2022/11/15/gsoc-report.html), the Scala Center is applying to participate GSoC 2023. We are currently reaching out to mentors to assemble a Project List for Scala GSoC 2023. This is a great opportunity for open source Scala project maintainers to get some fresh hands on their projects and help out new contributors at the very beginning of their Scala career!
+
+If you maintain an open source Scala project and you would like to mentor a student, please get in touch with us at [scala-gsoc@epfl.ch](mailto:scala-gsoc@epfl.ch) by the end of January 27th(Friday). We can't wait to hear from you!
```
Comment (SethTisue) [inline _posts/2023-01-23-gsoc-2023.md:12]: s/27th(Friday)/27th (Friday)/

File: _posts/2023-01-23-gsoc-2023.md (hunk 1 above)
Comment (yawen-guan) [reply _posts/2023-01-23-gsoc-2023.md]: done

File: _posts/2023-01-23-gsoc-2023.md (hunk 2 above)
Comment (yawen-guan) [reply _posts/2023-01-23-gsoc-2023.md]: Done. Added short intro "GSoC is an online mentorship program that matches beginner contributors to open source software development with mentors." according to https://youtu.be/7jD2tChhrWM?t=38

File: _posts/2023-01-23-gsoc-2023.md (hunk 3 above)
Comment (yawen-guan) [reply _posts/2023-01-23-gsoc-2023.md]: done

**Review by anatoliykmetyuk — APPROVED** [review]

**Pull request merged**